### ISSUE TITLE
IndexError in get_lngamma

### ISSUE THREAD

**Issue by Kycilak** [gustavochm/sgtpy#9]
Hello Gustavo,
I encountered an IndexError while trying to compute get_lngamma of a mixture with saftgammamie.
When I run this code:
```
from sgtpy import component, mixture, saftgammamie

comp1 = component(GC={'aCH':8,'C':4,'[CH2][OCH2]':1})
comp2 = component(GC={"CH3":2, "CH2":1, "COO":1})
mix = comp1+comp2
mix.saftgammamie()
eos = saftgammamie(mix)
eos.get_lngamma([0.1, 0.9], 320, 101325)
```
It throws the call stack with the bottom being:
```
File "/usr/local/lib/python3.10/dist-packages/sgtpy-0.0.15-py3.10-linux-x86_64.egg/sgtpy/gammamie_mixtures/ares.py", line 483, in d2ares_drho
    Dijklab[self.indexABij] *= Iijklab[self.indexAB_id]
IndexError: arrays used as indices must be of integer (or boolean) type

```
I know I am using the ether group without CH2 groups.
Do you know how I could make it compute?

Thank you very much for providing this resource and the support for it.
Have a nice day

Kycilak

**Comment by gustavochm** [comment]
Dear Kycilak,

Thanks a lot for catching this bug! From another fix related to the association configuration, the created array of indexes of association groups was cast to float64. I already fixed this and will update the version to PyPI.

By running your same sample code I got the following now:

<img width="631" alt="Screenshot 2022-11-14 at 16 57 54" src="https://user-images.githubusercontent.com/42041445/201753969-d020aac0-3030-4b06-9a1d-afa535205ad3.png">

Kind regards,
Gustavo
